lib/lotus/loader.rb

```diff
@@ -98,6 +99,25 @@ def _configure_view_framework!
       end
     end
 
+    def _configure_assets_framework!
+      configuration.assets_destination = Lotus.root.join('public', 'assets')
```

Comment:
@AlfonsoUceda Sadly yes, we need that temp variable `config`. If we try to access `configuration` from `duplicate`, it will reference `Lotus::Assets.configuration`, rather than the accessor that we have in `Loader`.

In other words:
```ruby
Lotus::Assets.duplicate(namespace) do
  puts configuration # => Lotus::Assets.configuration WRONG
  puts config # => Lotus::Configuration RIGHT
end
```